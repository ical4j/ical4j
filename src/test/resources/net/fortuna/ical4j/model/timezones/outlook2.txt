BEGIN:VCALENDAR
METHOD:REQUEST
VERSION:2.0
PRODID:-//Microsoft Corporation//Windows Live Calendar//EN
BEGIN:VTIMEZONE
TZID:Pacific Standard Time
BEGIN:STANDARD
DTSTART:20071104T020000
RRULE:FREQ=YEARLY;BYDAY=1SU;BYMONTH=11
TZOFFSETTO:-0800
TZOFFSETFROM:-0700
END:STANDARD
BEGIN:DAYLIGHT
DTSTART:20070311T020000
RRULE:FREQ=YEARLY;BYDAY=2SU;BYMONTH=3
TZOFFSETTO:-0700
TZOFFSETFROM:-0800
END:DAYLIGHT
END:VTIMEZONE
BEGIN:VEVENT
UID:181547ec-4781-453c-a50f-2064f46fa118
DTSTAMP:20140221T110411Z
CLASS:PUBLIC
X-MICROSOFT-CDO-BUSYSTATUS:TENTATIVE
TRANSP:OPAQUE
SEQUENCE:0
RRULE:FREQ=WEEKLY;COUNT=99;INTERVAL=1;BYDAY=MO,TU,WE,TH,FR;WKST=SU
DTSTART;TZID=Pacific Standard Time:20140221T150000
DTEND;TZID=Pacific Standard Time:20140221T160000
SUMMARY:Rec from outlook
DESCRIPTION:äöüß_<>\;„’’
LOCATION:äöüß_<>\;„’’
PRIORITY:0
EXDATE;TZID=Pacific Standard Time:20140221T150000
ATTENDEE;CUTYPE=INDIVIDUAL;CN=roxana.dragavei@web.de;ROLE=REQ-PARTICIPANT;PARTSTAT=NEEDS-ACTION;RSVP=TRUE:MAILTO:roxana.dragavei@web.de
ATTENDEE;CUTYPE=INDIVIDUAL;CN=roxana.dragavei@gmail.com;ROLE=REQ-PARTICIPANT;PARTSTAT=NEEDS-ACTION;RSVP=TRUE:MAILTO:roxana.dragavei@gmail.com
ORGANIZER;CN=Begenda T:MAILTO:begendatest01@outlook.com
END:VEVENT
END:VCALENDAR