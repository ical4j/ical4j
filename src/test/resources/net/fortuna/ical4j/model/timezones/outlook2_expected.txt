BEGIN:VCALENDAR
METHOD:REQUEST
VERSION:2.0
PRODID:-//Microsoft Corporation//Windows Live Calendar//EN
BEGIN:VTIMEZONE
TZID:America/Vancouver
BEGIN:STANDARD
DTSTART:20071104T020000
RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU
TZOFFSETTO:-0800
TZOFFSETFROM:-0700
END:STANDARD
BEGIN:DAYLIGHT
DTSTART:20070311T020000
RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=2SU
TZOFFSETTO:-0700
TZOFFSETFROM:-0800
END:DAYLIGHT
END:VTIMEZONE
BEGIN:VEVENT
UID:181547ec-4781-453c-a50f-2064f46fa118
DTSTAMP:20140221T110411Z
CLASS:PUBLIC
X-MICROSOFT-CDO-BUSYSTATUS:TENTATIVE
TRANSP:OPAQUE
SEQUENCE:0
RRULE:FREQ=WEEKLY;WKST=SU;COUNT=99;INTERVAL=1;BYDAY=MO,TU,WE,TH,FR
DTSTART;TZID=America/Vancouver:20140221T150000
DTEND;TZID=America/Vancouver:20140221T160000
SUMMARY:Rec from outlook
DESCRIPTION:äöüß_<>\;„’’
LOCATION:äöüß_<>\;„’’
PRIORITY:0
EXDATE;TZID=America/Vancouver:20140221T150000
ATTENDEE;CUTYPE=INDIVIDUAL;CN=roxana.dragavei@web.de;ROLE=REQ-PARTICIPANT;PARTSTAT=NEEDS-ACTION;RSVP=TRUE:MAILTO:roxana.dragavei@web.de
ATTENDEE;CUTYPE=INDIVIDUAL;CN=roxana.dragavei@gmail.com;ROLE=REQ-PARTICIPANT;PARTSTAT=NEEDS-ACTION;RSVP=TRUE:MAILTO:roxana.dragavei@gmail.com
ORGANIZER;CN=Begenda T:MAILTO:begendatest01@outlook.com
END:VEVENT
END:VCALENDAR